src/hooks/useLoadAccountData.ts

```diff
@@ -23,17 +22,16 @@ export default function useLoadAccountData() {
       // tokens + nfts
       if (network === networkTypes.mainnet) {
         const p1 = dispatch(dataLoadState());
-        const p2 = dispatch(uniqueTokensLoadState());
```

Comment:
no longer need to prefetch nfts bc unlike redux, react query will fetch the nfts from the interweb (or cache if available) on demand